app/models/dataset.rb

```diff
@@ -0,0 +1,18 @@
+class Dataset < ActiveRecord::Base
```

Comment:
Add this to ensure user is associated with create/update/delete. See other models (e.g. Audiorecording) for usage.

'# ensures that creator_id, updater_id, deleter_id are set
include UserChange

Reply:
ahh that's what we were missing